docs/docs/marketplace/marketplace_overview.md

```diff
@@ -0,0 +1,95 @@
+---
+id: marketplace-overview
+title: 'Marketplace : Overview'
+---
+
+# Marketplace : Overview
+
+With ToolJet Marketplace, ToolJet users can conveniently add custom plugins (datasources) to their workspaces. This feature enables users to create plugins that cater to their specific needs and integrate them seamlessly with ToolJet.
+
+<div style={{textAlign: 'center'}}>
+
+<img className="screenshot-full" src="/img/marketplace/overview/marketplace.png" alt="Marketplace" />
+
+</div>
+
+## Enabling the marketplace 
+
+To **Activate** the marketplace feature, users need to add the following environment variable to their **[`.env`](/docs/setup/env-vars#marketplace)** file:
```

Comment:
Using "activate" in this context could be misleading because the marketplace feature may already be installed or available, and the instruction is only for enabling it. Therefore, it would be better to use a word such as "enable" instead of "activate."

```suggestion
To **Enable** the marketplace feature, users need to add the following environment variable to their **[`.env`](/docs/setup/env-vars#marketplace)** file:
```